source/frontend/src/mocks/mockProjects.ts

```diff
@@ -33,22 +33,52 @@ export const mockProjects = (): Api_Project[] => [
 
 export const mockProjectPostResponse = (
   id: number = 1,
```

Comment:
Not sure we want to send Id = 1 to create a new project. I believe the existing pattern is to leave Id undefined for POSTs